troop/views.py

```diff
@@ -233,3 +237,60 @@ def form_invalid(self, form):
         response = super().form_invalid(form)
         response.status_code = 422  # Unprocessable Entity
         return response
+
+
+def csv_participant_export(request, troop_number):
```

Comment:
If we use a class-based view here we could group `participant_row` into this class :)

Reply:
I find it quite hard to find the things that I need in the django documentation to do this in a class-based way (I have the feeling that making it class-based adds a lot of complexity and magic for no benefit).

Can you show me a sample implementation? 

Besides, I was thinking that the `participant_row` method could be quite useful for other exports

Reply:
I think we could do it [like this](https://gist.github.com/MhmdRyhn/820e3277c157098655dd3ae117be4285). If you want to keep `participant_row` outside of the class that's fine of course.

Reply:
Done